src/plugins/intel_cpu/src/mkldnn_exec_network.cpp

```diff
@@ -227,13 +240,65 @@ InferenceEngine::Parameter MKLDNNExecNetwork::GetMetric(const std::string &name)
             configKeys.push_back(key.first);
         }
         IE_SET_METRIC_RETURN(SUPPORTED_CONFIG_KEYS, configKeys);
-    } else if (name == METRIC_KEY(OPTIMAL_NUMBER_OF_INFER_REQUESTS)) {
-        Config engConfig = GetGraph()._graph.getProperty();
-        auto option = engConfig._config.find(CONFIG_KEY(CPU_THROUGHPUT_STREAMS));
-        IE_ASSERT(option != engConfig._config.end());
-        auto streams = std::stoi(option->second);
-        IE_SET_METRIC_RETURN(OPTIMAL_NUMBER_OF_INFER_REQUESTS, static_cast<unsigned int>(
-            streams ? streams : 1));
+    }
+
+    // some params can be made RW with minor / major changes
+    if (name == ov::supported_properties) {
+        return std::vector<ov::PropertyName> {
+            RO_property(ov::model_name.name()),
+            RO_property(ov::optimal_number_of_infer_requests.name()),
+            RO_property(ov::streams::num.name()),// Was CONFIG CPU_THROUGHPUT_STREAMS
+            RO_property(ov::affinity.name()), // Was CONFIG CPU_BIND_THREAD
+            RO_property(ov::inference_num_threads.name()), // Was CONFIG CPU_THREADS_NUM
+            RO_property(ov::enable_profiling.name()), // @todo can be easely made RW (supposed to be RW). Was CONFIG
+            RO_property(ov::hint::inference_precision.name()), // was CONFIG ENFORCE_BF16
+            RO_property(ov::hint::performance_mode.name()), // Was CONFIG KEY_PERFORMANCE_HINT
+            RO_property(ov::hint::num_requests.name()), // Was CONFIG KEY_PERFORMANCE_HINT_NUM_REQUESTS
+            RO_property(ov::cache_dir.name()), // Was CONFIG, did nothing. Maybe we need to support it just in case (for future features)?
```

Comment:
@IvanNovoselov should it be really supported?

Reply:
We shouldn't return this as supported property.